packages/typescript/WORKSPACE

```diff
@@ -38,10 +38,11 @@ rules_nodejs_dev_dependencies()
 
 # We use git_repository since Renovate knows how to update it.
 # With http_archive it only sees releases/download/*.tar.gz urls
+# TODO(gregmagolan): switch back to upstream
 git_repository(
     name = "build_bazel_rules_typescript",
-    commit = "b1fa8ad9d15a4b46da800b33333512e538c69bd4",
-    remote = "http://github.com/bazelbuild/rules_typescript.git",
+    commit = "817e60741d25cb1ba61f54d32b488bea0c9dfabc",
```

Comment:
I assume you considered a `patch` here?

Reply:
The patch would be huge. I think this is the easier route.